changelog.txt

```diff
@@ -1,5 +1,6 @@
 Version 5.0.7 (TBD)
     Added highlighting of matched text in autocompleter items #TINY-3687
+    Added link_force_external_targets option to prepend http:// or https:// prefixes if the supplied link does not contain a protocol prefix #TINY-4335
```

Comment:
Feel free to add your username as a credit, similar to other github tickets (they're tagged `GH-####` in the changelog)